resources/views/admin.blade.php

```diff
@@ -70,7 +70,11 @@
                             <select id="select" name="company_id" class="form-select form-control m-2 " aria-label="Default select example">
                                 <option hidden>Company</option>
                                 @foreach ( $companies  as $company )
-                                    <option value="{{ $company->id }}"  class="create-option" data-branch="{{ $company->branch }}"> {{ $company->name }} </option>
+                                    @if ($company->trashed())
```

Comment:
please explain the purpose of this if else conditions

Reply:
With those if else conditions I'm checking whether a company was soft deleted or not...and if its soft deleted I'm adding a class named trash for that option only...so later below i can target that class and toggle visibility for edit and create forms  @Raza9798 #56

Reply:
the proper way to handle this cases. remove the if else directives. and add a conditional class by checking `deleted_at`. so  you can handle this in proper way @axnish27

Reply:
Okay got it...ill refactor this code